ml_genn/layers/fs_neurons.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+from pygenn.genn_model import create_dpf_class, create_custom_neuron_class
+from ml_genn.layers.neurons import Neurons
+
+fs_relu_first_phase_model = create_custom_neuron_class(
+    'fs_relu_first_phase',
+    param_names=['K', 'alpha'],
+    derived_params=[("scale", create_dpf_class(lambda pars, dt: pars[1] * 2**(-pars[0]))())],
+    var_name_types=[('Fx', 'scalar'), ('Vmem', 'scalar')],
+    sim_code='''
+    // Convert K to integer
+    const int kInt = (int)$(K);
+
+    // Get timestep within presentation
+    const int pipeTimestep = (int)($(t) / DT);
+    const int phaseTimestep = pipeTimestep % kInt;
+
+    // Calculate magic constants. For RelU hT=h=T
+    // **NOTE** d uses last timestep as that was when spike was SENT
+    const scalar hT = $(scale) * (1 << (kInt - (1 + phaseTimestep)));
+    const scalar d = $(scale) * (1 << ((kInt - phaseTimestep) % kInt));
```

Comment:
phaseTimestep < kInt, so the "% kInt" does nothing?

Reply:
I had to think about this again when you asked this...They only differ in the first timestep of the phase i.e. ``phaseTimestep=0`` when you want to be  using the ``d`` value for the last timestep of the previous phase. So, if ``kInt=10``, ``1 << (10 - 0) = 1024`` which incorrectly applies a very large scale whereas ``1 << ((kInt - 0) % 10) = 1`` which is correct